crates/pathfinder/src/rpc/v02/method/add_invoke_transaction.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Context;
+
+use crate::core::StarknetTransactionHash;
+use crate::rpc::v02::types::request::BroadcastedInvokeTransaction;
+use crate::rpc::v02::RpcContext;
+use crate::sequencer::ClientApi;
+
+crate::rpc::error::generate_rpc_error_subset!(AddInvokeTransactionError);
+
+#[derive(serde::Deserialize, Debug, PartialEq, Eq)]
+pub struct AddInvokeTransactionInput {
+    invoke_transaction: BroadcastedInvokeTransaction,
+}
+
+#[derive(serde::Serialize)]
+pub struct AddInvokeTransactionOutput {
+    transaction_hash: StarknetTransactionHash,
+}
+
+pub async fn add_invoke_transaction(
+    context: RpcContext,
+    input: AddInvokeTransactionInput,
+) -> Result<AddInvokeTransactionOutput, AddInvokeTransactionError> {
+    let response = match input.invoke_transaction {
+        BroadcastedInvokeTransaction::V0(v0) => context
+            .sequencer
+            .add_invoke_transaction(
+                v0.version,
+                v0.max_fee,
+                v0.signature,
+                Some(v0.nonce),
+                v0.contract_address,
+                Some(v0.entry_point_selector),
+                v0.calldata,
+            )
+            .await
+            .context("Sending V0 invoke transaction to gateway")?,
+        BroadcastedInvokeTransaction::V1(v1) => context
+            .sequencer
+            .add_invoke_transaction(
+                v1.version,
+                v1.max_fee,
+                v1.signature,
+                Some(v1.nonce),
+                v1.sender_address,
+                None,
```

Comment:
@kkovaacs and `entry point selector` is now deprecated?

Reply:
Yes, there's no `entry_point_selector` for v1 invoke transactions (the reasoning behind it is that it's always done through an account contract, so the only meaningful entry point is `__execute__`).